src/sh/python-lint/format.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+# cspell:words realpath, autoflake, isort, mypy
+
+POETRY_SUBDIRECTORY=./src/python/hooks
+
+cd $POETRY_SUBDIRECTORY || exit 1
```

Comment:
Any way this could cd back to repo root upon completion? When I try and run this using `source` I end up stuck in a different dir than when I started